packages/core/src/app/checkout/CheckoutStepStatus.ts

```diff
@@ -1,6 +1,7 @@
 import CheckoutStepType from './CheckoutStepType';
 
 export default interface CheckoutStepStatus {
+    activeStepType?: CheckoutStepType;
     isActive: boolean;
```

Comment:
A default step may have `isActive` as true, so we need to pass the `activeStepType` to determine the current active step.